src/sql/workbench/services/accountManagement/browser/accountManagementService.ts

```diff
@@ -241,11 +270,30 @@ export class AccountManagementService implements IAccountManagementService {
 				}
 			}
 
+			this._telemetryService.createActionEvent(TelemetryView.LinkedAccounts, TelemetryAction.RefreshAzureAccount)
+				.withAdditionalProperties({
+					'AuthenticationLibrary': this._authLibrary
```

Comment:
You use "AuthLibrary" in other places, but "AuthenticationLibrary" here. Maybe make a constant and use that everywhere instead so everything is consistent.